### PR TITLE
doc: add kitex v0.11.0 release log

### PR DESCRIPTION
**Pull request by HeyJavaBean**
#### What type of PR is this?

doc

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/zh/blog/releases/Kitex/release-v0_11_0.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+### 新特性
+1. **重试：新增混合重试功能**：支持同时开启「失败重试」+「Backup Request」两种策略，能够在降低长尾请求的同时提高请求的重试成功率，详见 [请求重试](https://www.cloudwego.io/zh/docs/kitex/tutorials/service-governance/retry/)
+2. **自定义 Payload 校验**：为避免硬件故障或数据篡改导致收发的数据不一致，Kitex 提供了对 Payload 报文的校验功能，并支持自定义扩展
+
+### 功能优化
+1. **Frugal ARM 性能优化**：Frugal 支持了基于反射的高性能编解码，升级到 Frugal v0.2.0 即可
+2. **Kitex Tool 代码生成提速**：提供了 -rapid 参数，可以无需安装 Thriftgo ，且速度略有提高。下个版本之后将作为默认行为。
+
+### 其他
+1. 支持版本 Go 1.18~1.23，最低支持变为 Go 1.18，如果你的 Go 版本过低，编译时会有提示：note: module requires Go 1.18
```
Comment (GuangmingLuo) [inline content/zh/blog/releases/Kitex/release-v0_11_0.md:20]: 可以使用代码块 `note: module requires Go 1.18`

File: content/zh/blog/releases/Kitex/release-v0_11_0.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## **重要变更介绍**
+
+### 新特性
+1. **重试：新增混合重试功能**：支持同时开启「失败重试」+「Backup Request」两种策略，能够在降低长尾请求的同时提高请求的重试成功率，详见 [请求重试](https://www.cloudwego.io/zh/docs/kitex/tutorials/service-governance/retry/)
+2. **自定义 Payload 校验**：为避免硬件故障或数据篡改导致收发的数据不一致，Kitex 提供了对 Payload 报文的校验功能，并支持自定义扩展
+
+### 功能优化
+1. **Frugal ARM 性能优化**：Frugal 支持了基于反射的高性能编解码，升级到 Frugal v0.2.0 即可
+2. **Kitex Tool 代码生成提速**：提供了 -rapid 参数，可以无需安装 Thriftgo ，且速度略有提高。下个版本之后将作为默认行为。
```
Comment (GuangmingLuo) [inline content/zh/blog/releases/Kitex/release-v0_11_0.md:17]: 可以使用代码块`-rapid`

File: content/zh/blog/releases/Kitex/release-v0_11_0.md (hunk 1 above)
Comment (HeyJavaBean) [reply content/zh/blog/releases/Kitex/release-v0_11_0.md]: fixed

File: content/zh/blog/releases/Kitex/release-v0_11_0.md (hunk 2 above)
Comment (HeyJavaBean) [reply content/zh/blog/releases/Kitex/release-v0_11_0.md]: fixed

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**